src/Zicht/Bundle/FrameworkExtraBundle/Resources/config/services.xml

```diff
@@ -29,6 +29,11 @@
             <tag name="form.type" alias="zicht_markup_type"/>
         </service>
 
+        <service id="Zicht\Bundle\FrameworkExtraBundle\Form\JsonSchemaType" class="Zicht\Bundle\FrameworkExtraBundle\Form\JsonSchemaType">
+            <argument>%kernel.root_dir%/../web</argument>
```

Comment:
Dit is tricky, dit moet `/public` zijn voor SF4 e.v. maar er is geen goede manier om dit op te halen. Voorheen hadden we onze eigen Kernel in de Symfony-Util bundle, die had de methode `getPublicDir` oid, maar die gebruiken we niet meer in SF4. 
Het enige dat ik kan terugvinden bij Symfony zelf is dit https://github.com/symfony/framework-bundle/blob/master/Command/AssetsInstallCommand.php#L261

Reply:
Ja, jij had in een ongerelateerde PR de Kernel geinjecteerd en daar `getWebDir` op aangeroepen, dat zal ik hier ook doen

Reply:
https://github.com/zicht/ticket-trigger-culture/pull/423/files

Aangepast.

Reply:
Nice. 

"Probleem" is alleen dat we waarschijnlijk toch weer terug moeten naar `<argument>%kernel.root_dir%/../public</argument>` of iets vergelijkbaars voor release/9.x omdat dit dus (kwam ik ook pas later achter) een methode is die wij zelf bedacht hebben in onze eigen Kernel uit die Util-bundle :weary:

Reply:
Wil je daar nu nog iets mee of pakken we dat op als we dit naar 9.x mergen?

Reply:
Dan liever `%kernel.root_dir%/../web` houden en dit bij 9.x aanpassen naar `%kernel.root_dir%/../public`.  Dat was wat ik oorspronkelijk in gedachten had.

Maar is het dan ook niet handig om dat te doen bij https://github.com/zicht/ticket-trigger-culture/pull/423/files?  Dan doen we het iig consistent...

Reply:
Ok. ik zie ook geen andere manier :+1: 

Ik pak dat in TTC wel mee als we daar de SF4 upgrade gaan doen.

Reply:
Check.  De code is weer naar de originele `<argument>%kernel.root_dir%/../web</argument>`